python/guide/packages/gluon/trainer.md

```diff
@@ -1,72 +1,95 @@
 # Trainer
 
-A trainer updates neural network parameters by an optimization
-method.
+Training a neural network model consists of iteratively performing three simple steps. 
 
-## Basic Usages
+The first step is the forward step which computes the loss.  In MXNet gluon, this first step is achieved by doing a forward pass by calling `net.forward(X)` or simply `net(X)` and then calling the loss function with the result of the forward pass and the labels. For example `l = loss_fn(net(X), y)`.
 
-We first create a simple Perceptron and run forward and
-backward passes to obtain the gradients.
+The second step is the backward step which computes the gradient of the loss with respect to the parameters. In gluon, this step is  achieved by doing the first step in an `autograd.record()` scope to record the computations needed to calculate the loss, and then calling `l.backward()` to compute the gradient of the loss with respect to the parameters.
```

Comment:
Add link to autograd tutorial somewhere here.

Reply:
done in next commit

Reply:
will do.